tools/flang1/flang1exe/main.c

```diff
@@ -94,6 +94,7 @@ static const char *feature = "pgfortran";
 static const char *feature = "pgfortran";
 #endif
 
+bool g_list_macros = FALSE;
```

Comment:
Please declare it (with `extern`) in main.h and include `"main.h"` in both fpp.c and accpp.c

Reply:
Actually, avoid main.h. As I looked into it, I've found it terribly messed up. Use global.h instead, it is already included by fpp.c and accpp.c